src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/DataFrameReaderTests.cs

```diff
@@ -1,72 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Collections.Generic;
```

Comment:
why is this file deleted ?

Reply:
This is not related to UDF test. This is for DataFrameReaderTest.

Reply:
This whole file is deleted.  It shouldn't be.

Reply:
Added!